packages/sdk/src/nftx/types.ts

```diff
@@ -0,0 +1,10 @@
+export type OrderParams = {
```

Comment:
We should keep the list of incremental prices in the order object: https://github.com/reservoirprotocol/core/blob/e2e774b3b1ccc8532de0ac664bf8c16f69617a09/packages/sdk/src/sudoswap/order.ts#L29